packages/system/src/Item/api/operations.graphql

```diff
@@ -88,3 +119,54 @@ query itemsListView(
     }
   }
 }
+
+query itemsWithStats($storeId: String!) {
+  items(storeId: $storeId) {
+    ... on ItemConnector {
+      __typename
+      nodes {
+        __typename
+        code
+        id
+        isVisible
+        name
+        unitName
+        stats(storeId: $storeId) {
+          __typename
+          averageMonthlyConsumption
+          availableStockOnHand
+          availableMonthsOfStockOnHand
+        }
+      }
+    }
+  }
+}
+
+query itemById($storeId: String!, $itemId: String!) {
```

Comment:
added this for searching for a single item (which is used for getting stock lines..)